app/models/individual_market/attestation.rb

```diff
@@ -0,0 +1,31 @@
+# frozen_string_literal: true
+
+module IndividualMarket
+  # Represents a collection of attestations made by an applicant during the enrollment process
```

Comment:
Can an admin attest on behalf of a user?  Do we need to track that distinction somewhere?  The wording in here implies that it's always the account holder who is the user attesting to something.